vmcnet/train/default_config.py

```diff
@@ -346,6 +346,40 @@ def get_default_vmc_config() -> Dict:
                 "learning_rate": 5e-2,  # needs to be tuned with everything else
                 "learning_decay_rate": 1e-4,
             },
+            # >>>>>>>>>> from gg-min-sr-mom >>>>>>>>>>
+            "proxsr": {
+                # Learning rate params
+                "schedule_type": "inverse_time",  # constant or inverse_time
+                "learning_rate": 5e-2,  # needs to be tuned with everything else
+                "learning_decay_rate": 1e-4,
+                # ProxSR-specific params
+                "complement_decay": 0.95,
```

Comment:
Make default 0.99 instead of 0.95?

Reply:
Maybe rename to prev_grad_decay? I'm not sure that's that clear either but complement_decay seems to have made more sene before when we had all three options.